packages/cli/src/index.ts

```diff
@@ -16,20 +16,24 @@ program
 
 program
   .command('dev')
-  .description('Start a Novu Dev Studio and a localtunnel')
-  .option('-p, --port <value>', 'Set the local port for the Novu endpoint, defaults to 4000', '4000')
-  .option('-r, --route <value>', 'Set the Novu Endpoint mounted route, defaults to /api/novu', '/api/novu')
-  .option('-o, --origin <value>', 'Set the origin for the Novu endpoint')
-  .option('-sp, --studio-port <value>', 'Set the local port for the Novu Local Studio server, defaults to 2022', '2022')
-  .option(
-    '-so, --studio-remote-origin <value>',
-    'Set the remote origin for Novu Studio, used for staging environment and local development, defaults to https://web.novu.co',
-    'https://web.novu.co'
-  )
-  .option(
-    '--region <value>',
-    'Studio Origin SPA, used for staging environment and local development, defaults to us',
-    'us'
+  .description(
+    `Start Novu Studio and a localtunnel
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ localtunnel is a named tool, writing it as "local tunnel" makes it more generic and immune to any changes in tooling we do down the line

Reply:
100%. Will update.